libs/blocks/global-navigation/global-navigation.js

```diff
@@ -63,6 +64,13 @@ const CONFIG = {
             config: {
               enableLocalSection: true,
               miniAppContext: {
+                onMessage: (name, payload) => {
+                  if (name === 'System' && payload.subType === 'AppInitiated') {
+                    window.adobeProfile?.getUserProfile()
```

Comment:
Just curious, what is this `adobeProfile` object on `window` and where is that set? I'm curious what exact profile is fetched from where. The PR description says that IMS and cc-collab are not being used anymore.

Reply:
Thanks @overmyheadandbody for the additional details 👍